lib/fshash/metadata.go

```diff
@@ -0,0 +1,118 @@
+package fshash
+
+import (
+	"archive/tar"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/spacemonkeygo/errors"
+	"github.com/ugorji/go/codec"
+	"polydawn.net/repeatr/def"
+)
+
+type Metadata tar.Header
```

Comment:
Absolutely.  And effectively, we kind of do this -- in the marshal method.  This struct might be deserving of replacement with something more "ours" in the future, totally open to that at the first sign that keeping it becomes an irritation.